cmd/root.go

```diff
@@ -56,16 +61,23 @@ func NewRootCmd() *cobra.Command {
 					fmt.Sprintf("net.netfilter.nf_conntrack_tcp_timeout_close_wait=%d", options.TimeoutCloseWaitSecs),
 				)
 				out, err := sysctl.CombinedOutput()
-				log.Println(string(out))
 				if err != nil {
+					log.Errorln(string(out))
```

Comment:
It's unclear to me why we'd sometimes log with `Errorln` and sometimes with `Error`.  I assume the logger automatically formats log messages one per line so I don't think we ever need to use function with explicitly adds newlines like this one.

Reply:
My bad. Fixing it.